models/public/ctpn/ctpn.md

```diff
@@ -0,0 +1,82 @@
+# ctpn
+
+## Use Case and High-Level Description
+
+Detecting Text in Natural Image with Connectionist Text Proposal Network. For details see [paper](https://arxiv.org/pdf/1609.03605.pdf).
+
+## Example
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| Type                            | Object detection                          |
+| GFlops                          | 55.813                                    |
+| MParams                         | 17.237                                    |
+| Source framework                | Tensorflow\*                              |
+
+## Performance
+
+## Input
+
+### Original model
+
+Image, name - `image_tensor`, shape - [1x300x300x3], format [BxHxWxC],
+   where:
+
+    - B - batch size
+    - H - image height
+    - W - image width
+    - C - number of channels
+
+   Expected color order - RGB.
+
```

Comment:
in the config:
- --mean_values=Placeholder[102.9801,115.9465,122.7717]
missing here?